packages/metablocks/src/api.ts

```diff
@@ -21,6 +21,7 @@ import {
   UniverseApiArgs,
   WithdrawNftApiArgs,
 } from "./types/types";
+import { SendTxRequest } from "@project-serum/anchor/dist/cjs/provider";
```

Comment:
why are you referring the dist path ?

Reply:
done, changed to use custom types, It was auto added by VScode